src/hooks/xcm/SubstrateApi.ts

```diff
@@ -116,13 +116,22 @@ class ChainApi {
 
   public async getBalance(address: string) {
     try {
+      await this._api?.isReady;
       return ((await this._api.query.system.account(address)) as any).data.free.toBn() as BN;
     } catch (e) {
       console.error(e);
       return new BN(0);
     }
   }
 
+  public async isReady(): Promise<void> {
```

Comment:
![Screen Shot 2022-05-17 at 2 09 48 PM](https://user-images.githubusercontent.com/40356749/168808087-83d6f422-9519-4f59-8d67-e4b6def12734.png)

I tried to transfer right after I XCMed DOT, but I'm getting an RPC connection error. It seems like the portal failed to reconnect to Astar Network after finishing the XCM transfer from the relaychain

Reply:
Transfer feature has not implemented yet. @bobo-k2 is working on it.